src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.iOSmacOS.cs

```diff
@@ -103,7 +103,15 @@ private static IDisposable InnerCreateLayer(UIElement owner, CALayer parent, Lay
 
 			if (cornerRadius != CornerRadius.None)
 			{
-				var maxInnerRadius = Math.Max(0, Math.Min((float)area.Width / 2 - widthOffset, (float)area.Height / 2 - heightOffset));
+				var maxOuterRadius = Math.Max(0, Math.Min((float)area.Width / 2 - widthOffset, (float)area.Height / 2 - heightOffset));
```

Comment:
move `area.Width / 2` and `area.Height / 2`  to variables to avoid double computation